packages/openid4vc/src/shared/issuerMetadataUtils.ts

```diff
@@ -23,16 +27,88 @@ export function getTypesFromCredentialSupported(credentialSupported: OpenId4VciC
   throw Error(`Unable to extract types from credentials supported. Unknown format ${credentialSupported.format}`)
 }
 
+export function credentialConfigurationSupportedToCredentialSupported(
+  config: CredentialConfigurationSupportedV1_0_13
+): OpenId4VciCredentialSupportedWithId {
+  return {
+    id: config.id as string,
+    format: config.format,
+    scope: config.scope,
+    cryptographic_binding_methods_supported: config.cryptographic_binding_methods_supported,
+    // @ts-expect-error this property was removed for some reason
+    cryptographic_suites_supported: config.credential_signing_alg_values_supported,
+    proof_types_supported: config.proof_types_supported,
+    display: config.display,
+    ...(config.vct && { vct: config.vct }),
+    claims: config.format === 'vc+sd-jwt' ? config.claims : undefined,
+    credentialSubject: config.format !== 'vc+sd-jwt' ? config.credential_definition.credentialSubject : undefined,
+    types: config.format !== 'vc+sd-jwt' ? config.credential_definition.type : undefined,
+  }
+}
+
+export function credentialSupportedToCredentialConfigurationSupported(
+  credentialSupported: OpenId4VciCredentialSupportedWithId
+): CredentialConfigurationSupportedV1_0_13 {
+  return {
+    vct: credentialSupported.format === 'vc+sd-jwt' ? credentialSupported.vct : undefined,
+    id: credentialSupported.id,
```

Comment:
* TODO: These two (vct and id) are solely added because of backward compatibility with sd-jwt. as soons as we have a clear understanding of the new sd-jwt issuer protocol we can remove this
   
   that's why I added them

Reply:
What is the new sd jwt issuer protocol?

Reply:
it is a comment from sphereon not sure what they mean